### DISCUSSION TITLE
background does not go all the way to the bottom

### DISCUSSION THREAD

**Discussion by Timo-Thees** [neuefische/questions#440] [Web] [answered]
Straight forward, I have a <div> that should work as my background image. It is wrapped around all my components and is styled like this:
`const Body = styled.div`
  background: #eeeeee;
  background-attachment: fixed;
  width: 100%;
  height: 100%;
  top: 0px;
  right: 0px;
  bottom: 0px;
  left: 0px;
`;`
However, on a page with only 2 objects in it it does not fill the screen. How can I make it?

**Comment by actyralf** [comment, marked as answer]
I guess `min-height: 100vh` would help. To prevent scrolling on mobile devices, have a look at this: https://dev.to/fenok/stretching-body-to-full-viewport-height-the-missing-way-2ghd#:~:text=Applying%20min%2Dheight%3A%20100vh%20to,grow%20even%20more%20if%20necessary.

**Reply by Timo-Thees** [reply]
Thank you, it works!
